ProductService.js

```diff
@@ -0,0 +1,136 @@
+async function getProductList(page, pageSize, keyword) {
+  let data;
+  try {
+    const params = { page, pageSize, keyword };
+    const url = new URL("https://panda-market-api-crud.vercel.app/products");
+    Object.keys(params).forEach((key) => {
+      if (params[key]) {
+        url.searchParams.append(key, params[key]);
+      }
+    });
+
+    const res = await fetch(url);
+    if (!res.ok) {
+      throw new Error(`HTTP error! ${res.status}`);
+    }
+
+    const data = await res.json();
+    return data;
+  } catch (error) {
+    console.error("Error!");
+  } finally {
+    console.log("Finished");
+  }
+}
+
+async function getProduct(id) {
+  try {
+    const res = await fetch(
+      `https://panda-market-api-crud.vercel.app/products/${id}`
+    );
+
+    if (!res.ok) {
+      throw new Error(`HTTP error! ${res.status}`);
+    }
+
+    const data = await res.json();
+    return data;
+  } catch (error) {
+    console.log("Error!");
+  } finally {
+    console.log("Finished!");
+  }
+}
+
+async function createProduct(name, description, price, tags, images) {
+  try {
+    const res = await fetch(
+      "https://panda-market-api-crud.vercel.app/products",
+      {
+        method: "POST",
+        body: JSON.stringify({
+          name: name,
+          description: description,
+          price: price,
+          tags: tags || [],
+          images: images || [],
```

Comment:
기본 값을 잘 준 것 같습니다.
아래와 같이 ??을 사용할 수도 있는데, ||와 ??의 차이가 무엇인지 알아봐도 좋을 것 같습니다.
```suggestion
          tags: tags ?? [],
          images: images ?? [],
```